code/modules/research/techweb/all_nodes.dm

```diff
@@ -896,10 +896,10 @@
 	id = "nanite_hazard"
 	display_name = "Hazard Nanite Programs"
 	description = "Extremely advanced Nanite programs with the potential of being extremely dangerous."
-	prereq_ids = list("nanite_harmonic", "alientech")
+	prereq_ids = list("nanite_harmonic", "syndicate_basic")
 	design_ids = list("spreading_nanites","mindcontrol_nanites","mitosis_nanites")
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 10000)
-	export_price = 15000
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 15000)
+	export_price = 20000

```

Comment:
so are you going to remove this, orrrr?

Reply:
how do i? i have no idea how github works. and that's the reason this my old pr got here